test/CppTests/Tests/ExceptionTests.cs

```diff
@@ -281,6 +281,94 @@ public void RaisedReThrowException(ITestSettings settings)
             }
         }
 
+        [Theory]
```

Comment:
>Theory [](http://example.com/codeflow?start=9&length=6)

Disable new tests for LLDB?

Reply:
Oops, forgot if this was supported on macOS. 

I'll have to update the capabilities too

Reply:
Are these tests valid for VsDbg?

Reply:
VsDbg also calls it `all` and has the same syntax (+ some more stuff), so it seems like it should work (famous last words).

Reply:
Also there seems to be a bug with exceptions with the MinGW GDB version on the CI Windows machine.

> &(did not find exception probe (does libstdcxx have SDT probes?)

Reply:
It might not be in GDB but rather the version of libstdcxx that MinGW uses.

Do we output a reasonable warning when this happens?

Reply:
It's seems to be an output from GDB when it tries to evaluate the exception breakpoint and we forward it on to the UI as a log message.